jena-fuseki2/jena-fuseki-access/pom.xml

```diff
@@ -47,6 +47,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>pl.pragmatists</groupId>
```

Comment:
I wrote the unit tests first, before adding integration ones. I can see that the integration tests use a for-loop to simulate parameterisation - I'm happy to change this and remove this extra (Apache 2.0P) depedency.

Reply:
Or use JUnit5 which has this feature. JUnit4 and JUnit5 can be mixed.

Reply:
Ok, I'll update to use JUnit5 (once it's clear whether this is the right path for the proposed feature)